pokemon_random.py

```diff
@@ -0,0 +1,8 @@
+from constants import min_id, maximum_id
```

Comment:
Creating a file for this seems more random than `pokemon_random`, but IDK, JSIG (Just me, I guess)